controller_report_xls/controllers/main.py

```diff
@@ -1,27 +1,4 @@
 # -*- coding: utf-8 -*-
-###########################################################################
```

Comment:
Please add the short license format: https://github.com/OCA/maintainer-tools/blob/master/template/module/controllers/main.py#L1-L3